Packs/CommonScripts/ReleaseNotes/1_10_9.md

```diff
@@ -0,0 +1,6 @@
+#### Scripts
+##### ExtractEmailV2
+- Updated the Docker image to: *demisto/python3:3.10.8.37753*.
+- Fixed an issue of emails containing parts of encoded unicode points.
+- Updated the Email formatter.
+- Updated the tests for the changed formatter.
```

Comment:
Remove tests part

Reply:
Its part of the pack, and it was changed a lot, if someone will try to find it I want them to find this PR.